buildlib/pr/io_demo/io-demo.yml

```diff
@@ -17,12 +17,36 @@ parameters:
   - name: tests
     type: object
     default:
-      tag:
+      tag_cx4_rc:
         args: ""
         duration: 480
-      active:
+        interface: $(roce_iface_cx4)
+        tls: "rc_x"
+      active_cx4_rc:
         args: "-q -A"
         duration: 480
+        interface: $(roce_iface_cx4)
+        tls: "rc_x"
+      tag_cx6_rc:
+        args: ""
+        duration: 480
+        interface: $(roce_iface_cx6)
+        tls: "rc_x"
+      active_cx6_rc:
```

Comment:
1. Pls use "am" instead of "active" in the names: "am_cx6_rc"
2. Is it possible to set displayName, like this: 
- "Active messages on CX6/RC"
- "Active messages on CX6/DC"
- "Tag match on CX6/RC"
- "Tag match on CX6/DC"
- "Active messages on CX4"
- "Tag match on CX4"